data_safe_haven/config/__init__.py

```diff
@@ -1,5 +1,5 @@
-from .config import Config
 from .backend_settings import BackendSettings
+from .config import Config
```

Comment:
```suggestion
from data_safe_haven.config.backend_settings import BackendSettings
from .data_safe_haven.config.config import Config
```

?

Reply:
I think there are the same, this file is in the `data_safe_haven.commands` module. And we are allowing relative imports of siblings now.